src/ducks/commons/Nav.jsx

```diff
@@ -28,32 +28,33 @@ const navLinkMatch = props =>
  * to have a faster change of active (not waiting for the route to completely
  * change).
  */
-export const NavLink = React.memo(
-  withRouter(props => {
-    const {
-      children,
-      to,
-      clickState: [lastClicked, setLastClicked]
-    } = props
-    return (
-      <a
-        style={{ outline: 'none' }}
-        onClick={() => setLastClicked(to)}
-        href={`#/${to}`}
-        className={cx(
-          UINavLink.className,
-          (!lastClicked && navLinkMatch(props)) || lastClicked === to
-            ? UINavLink.activeClassName
-            : null
-        )}
-      >
-        {children}
-      </a>
-    )
-  })
-)
+export const NavLink = withRouter(props => {
+  const {
+    children,
+    to,
+    clickState: [lastClicked, setLastClicked]
+  } = props
+  return (
+    <a
+      style={{ outline: 'none' }}
+      onClick={() => setLastClicked(to)}
+      href={`#/${to}`}
+      className={cx(
+        UINavLink.className,
+        (!lastClicked && navLinkMatch(props)) || lastClicked === to
+          ? UINavLink.activeClassName
+          : null
+      )}
+    >
+      {children}
+    </a>
+  )
+})
```

Comment:
OK Can you just try to put the react memo *under* the withRouter ? If it works, good, otherwise LGTM :)

Reply:
It doesn't work :(